README.md

```diff
@@ -37,9 +37,9 @@ for those who like living dangerously.
 To host your own copy of Element, the quickest bet is to use a pre-built
 released version of Element:
 
-1. Download the latest version from https://github.com/vector-im/riot-web/releases
+1. Download the latest version from https://github.com/vector-im/element-web/releases
 1. Untar the tarball on your web server
-1. Move (or symlink) the `riot-x.x.x` directory to an appropriate name
+1. Move (or symlink) the `element-x.x.x` directory to an appropriate name
```

Comment:
This has not been changed yet AFAIK, see https://github.com/vector-im/element-web/issues/14896.

Reply:
Ok, i reverted this. In favor of changing it after #14896 landed.